src/sentry/middleware/access_log.py

```diff
@@ -125,8 +127,9 @@ def middleware(request: Request) -> Response:
         if not settings.LOG_API_ACCESS:
             return get_response(request)
 
-        if request.path_info.startswith(settings.ANONYMOUS_STATIC_PREFIXES):
+        if request.path_info.startswith(EXCLUSION_PATHS):
```

Comment:
TIL that `startswith` can takes tuple of prefixes.

Reply:
😅 same